src/cswinrt/strings/ComInteropHelpers.cs

```diff
@@ -127,7 +139,7 @@ namespace Windows.ApplicationModel.DataTransfer.DragDrop.Core
 {
 #if UAC_VERSION_1
 #if NET
-    [global::System.Runtime.Versioning.SupportedOSPlatform("Windows10.0.10240.0")]
+    [global::System.Runtime.Versioning.SupportedOSPlatform("Windows10.0.22000.0")]

```

Comment:
Why are these lines changed?

Reply:
> Why are these lines changed?

![Screenshot_2024-03-07-18-51-03-312_mark via](https://github.com/microsoft/CsWinRT/assets/49179966/0458fb19-eb7e-4d87-ab9e-5d6dab9f8797)

Because I found that some interfaces have a minimum version limit, referring to the Microsoft documentation, if the version is still 10240, it will cause misuse

Reply:
and some api contract is update to 19（26072），so add uac_version to 19
![Screenshot_2024-03-07-19-03-18-811_mark via](https://github.com/microsoft/CsWinRT/assets/49179966/133eaee3-bc26-49ce-b320-fb4d615d6d05)

Reply:
I think changes that are unrelated to the scope of the PR (in this case, `DataTransferManager`) should be avoided. You can probably submit another PR.

Reply:
> or

Uh, all right. Then I will close this PR and write a new PR